x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_fetch_browser_fields_capabilities.tsx

```diff
@@ -49,7 +54,14 @@ export const useFetchBrowserFieldCapabilities = ({
   }, [featureIds, http, toasts]);
 
   useEffect(() => {
-    if (isLoading !== undefined || featureIds.includes(INVALID_FEATURE_ID)) return;
+    if (initialBrowserFields && !isEmptyObject(initialBrowserFields)) {
+      setBrowserFields(initialBrowserFields);
+      return;
+    }
+
+    if (isLoading !== undefined || featureIds.includes(INVALID_FEATURE_ID)) {
```

Comment:
Okay great. that is better.

Reply:
@jcger, actually I will keep it as it is for now. Currently, siem has 2 views . In one of the view, we do not pass browserFields because we do not want to show the browserfields. In that we do not want system to query browserfields.

So for now I will keep it as it is. We can think what could be the best way around it.